src/components/interface/PortfolioHeader/PortfolioHeader.tsx

```diff
@@ -34,21 +19,11 @@ const PortfolioHeader = ({
 }: PortfolioHeaderProps) => {
   return (
     <>
-      <Box sx={{ textTransform: 'uppercase' }}>
-        <Typography variant="body2" sx={labelStyles}>
-          Net notional
-        </Typography>
-        <Typography variant="h1" sx={titleStyles}>
-          {isUndefined(portfolioData.totalNotional) && <>Loading...</>}
-          {!isUndefined(portfolioData.totalNotional) && (
-            <>
-              {currencySymbol}
-              {formatCurrency(portfolioData.totalNotional)} {currencyCode}
-            </>
-          )}
-        </Typography>
-      </Box>
-
+      <NetNotional
```

Comment:
nice, good idea to move it into a separate component

Reply:
yep yep more control if it is smaller segment ;)